tectonic/dpx-truetype.c

```diff
@@ -949,7 +949,7 @@ pdf_font_load_truetype (pdf_font *font)
         return  -1;
     }
 
-#if  ENABLE_NOEMBED
+#ifdef  ENABLE_NOEMBED
```

Comment:
Hmmm, the same consideration may apply here, but I don't know what `ENABLE_NOEMBED` does, so I'm not sure whether we want to act as if it is defined, or not. Out of scope for this PR, though.